### PR TITLE
Voice of God can no longer cause people to deathgasp

### PR DESCRIPTION
**Pull request by ChangelingRain**
I swear I left a comment about this on the original PR. Strange...

### CODE REVIEW

File: code/modules/surgery/organs/vocal_cords.dm [hunk 1]
```diff
@@ -567,12 +563,8 @@
 	else
 		cooldown = COOLDOWN_NONE
 
-	message_admins("[key_name_admin(user)] has said '[log_message]' with a Voice of God, affecting [english_list(listeners)], with a power multiplier of [power_multiplier].")
```
Comment (kevinz000) [inline code/modules/surgery/organs/vocal_cords.dm]: this not needed anymore?

**Comment by Repukan** [comment]
freeze

**Comment by PKPenguin321** [comment]
reason for change?

**Comment by Repukan** [comment]
https://tgstation13.org/phpBB/viewtopic.php?f=10&t=12457&view=unread#unread

This and Kev's PR

**Comment by PKPenguin321** [comment]
this is hilarious and will honestly only happen 1 in maybe a few thousand rounds, if ever again

it basically requires:
1. the ops to declare war rather than not
2. the ops to be slow as fuck despite declaring rush mode
3. robust miner(s) to be on the server
4. boss to spawn in relatively easy to get to locations
5. the miner(s) to find the specific boss in time
6. the miner(s) to actually defeat the boss and get the loot without dying
7. a miner to take the VoG to the medbay
8. another person existing that is competent and willing to do surgery
9. the surgery to be done in full and the miner back on his feet before the ops have arrived

the odds are tiny and this is a massive collision of bizarre chance not worthy of removing

a rare case of legitimate emergent gameplay that's really funny and wont be common enough to be problematic

**Comment by Supermichael777** [comment]
on the one hand that's awful

but lol

**Comment by AnturK** [comment]
❄️ Kor can reopen if he feels this is important enough.

**Comment by KorPhaeron** [comment]
Either this or the one @kevinz000 made should be merged this is funny once but it would be horrendous if it became the meta to just instagib ops

**Comment by PKPenguin321** [comment]
it cannot be the meta by virtue of the planets aligning being required to do it

**Comment by KorPhaeron** [comment]
Declaring war while Repukan is online is only two required steps

**Comment by Repukan** [comment]
Remove Repukan

**Comment by Repukan** [comment]
Edge case tbh

**Comment by Armhulen** [comment]
lol reopen own pr haha i ded

one person has done this, ever. the colossus has to be close to the station. the med doc has to implant it as well.

**Comment by Cyberboss** [comment]
Emergent imho

**Comment by XDTM** [comment]
Why are op implants triggered by deathgasp anyway, they already have an action button, suicide, succumb, whisper to do it

**Comment by Cyberboss** [comment]
Edge case af

**Pull request closed without merging**